src/main/resources/db/changelog/ddl/0012_supporting_information_non_mandatory.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="matt.wills@valtech.com" id="0011_EIP1-3803_alter_print_request - make supporting_information_format nullable" context="ddl">
+
+        <dropNotNullConstraint tableName="print_request" columnName="supporting_information_format" columnDataType="varchar(20)"/>
```

Comment:
I was hoping we didn't need to remove `supporting_information_format` as well, but Arron confirmed we do so I've included it in this PR.